warehouse/query-core/src/test/java/datawave/query/testframework/AccumuloSetup.java

```diff
@@ -0,0 +1,240 @@
+     * @return connector to Accumulo
+     * @throws AccumuloException
+     *             , AccumuloSecurityException, IOException, InterruptedException, TableExistsException, TableNotFoundException Accumulo error conditions
+     */
+    public Connector loadTables(final Logger parentLog, final RebuildingScannerTestHelper.TEARDOWN teardown, RebuildingScannerTestHelper.INTERRUPT interrupt)
+                    throws AccumuloException, AccumuloSecurityException, IOException, InterruptedException, TableExistsException, TableNotFoundException,
+                    URISyntaxException {
+        log.debug("------------- loadTables -------------");
+        
+        if (this.fileFormat != FileType.GROUPING) {
+            Assert.assertFalse("shard ids have not been specified", this.shardIds.isEmpty());
+            Assert.assertFalse("data types have not been specified", this.dataTypes.isEmpty());
+        }
+        
+        QueryTestTableHelper tableHelper = new QueryTestTableHelper(AccumuloSetup.class.getName(), parentLog, teardown, interrupt);
+        final Connector connector = tableHelper.connector;
+        tableHelper.configureTables(this.recordWriter);
+        
+        for (DataTypeHadoopConfig dt : this.dataTypes) {
+            HadoopTestConfiguration hadoopConfig = new HadoopTestConfiguration(dt);
+            TestFileLoader loader;
+            if (this.fileFormat.hasFileLoader()) {
+                loader = this.fileFormat.getFileLoader(dt.getIngestFile(), hadoopConfig);
+                ingestTestData(hadoopConfig, loader);
+            }
+        }
+        
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.METADATA_TABLE_NAME);
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.SHARD_TABLE_NAME);
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.SHARD_INDEX_TABLE_NAME);
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.SHARD_RINDEX_TABLE_NAME);
+        
+        // TODO: elsewhere?
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.FACET_TABLE_NAME);
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.FACET_METADATA_TABLE_NAME);
+        PrintUtility.printTable(connector, AbstractDataTypeConfig.getTestAuths(), QueryTestTableHelper.FACET_HASH_TABLE_NAME);
+        
+        return connector;
+    }
+    
+    private void ingestTestData(Configuration conf, TestFileLoader loader) throws IOException, InterruptedException {
+        log.debug("------------- ingestTestData -------------");
+        
+        Path tmpPath = new Path(tempFolder.toURI());
+        // To prevent periodic test cases failing, added "---" prefix for UUDID for test cases to support queries with _ANYFIELD_ starting with particular
```

Comment:
```suggestion
        // To prevent periodic test cases failing, added "---" prefix for UUID for test cases to support queries with _ANYFIELD_ starting with particular
```